core/res/configuration.toml

```diff
@@ -40,9 +40,9 @@ server = "localhost"
 port = "8200"
 healthcheckpath = "v1/sys/health"
 certpath = "v1/secret/edgex/pki/tls/edgex-kong"
-tokenpath = "res\\resp-init.json"
+tokenpath = "res\\kong-token.json"
```

Comment:
if you change this here we will need to make another change in edgex-go to fix the sed expression here: https://github.com/edgexfoundry/edgex-go/blob/edinburgh/snap/snapcraft.yaml#L977 so I'd prefer if you just change the configuration-docker.toml to unblock the docker deployment as the snap deployment is already fixed

Reply:
no problem. i will switch it back.